WcaOnRails/app/mailers/registrations_mailer.rb

```diff
@@ -61,7 +61,7 @@ def notify_registrant_of_deleted_registration(registration)
     @registration = registration
     mail(
       to: registration.email,
-      reply_to: registration.competition.managers.map(&:email),
+      reply_to: registration.competition.organizers.empty? ? registration.competition.delegates.map(&:email) : registration.competition.organizers.map(&:email),
       subject: "Your registration for #{registration.competition.name} has been deleted",
```

Comment:
The main reason for adding the method in the Competition model was to clean this file up. Please make use of `organizers_or_delegates`.